pkg/transaction/backend.go

```diff
@@ -75,7 +75,7 @@ func WaitSynced(ctx context.Context, logger log.Logger, backend Backend, maxDela
 			return nil
 		}
 
-		logger.Info("still waiting for Ethereum to sync", "block_time", blockTime)
+		logger.Info("still waiting for Gnosis to sync", "block_time", blockTime)
```

Comment:
```suggestion
		logger.Info("still waiting for blockchain RPC to sync", "block_time", blockTime)
```

Is there a reason to be firm on which chain this is? It could be sepolia for example.